Document/0x06b-Basic-Security-Testing.md

```diff
@@ -10,21 +10,56 @@ Although you can use a Linux or Windows host computer for testing, you'll find t
 
 The following is the most basic iOS app testing setup:
 
-- Ideally macOS host computer with admin rights.
-  - [Xcode](0x08-Testing-Tools.md#xcode) and [Xcode Command Line Tools](0x08-Testing-Tools.md#xcode-command-line-tools).
-  - An interception proxy tool such as [Burp Suite](0x08-Testing-Tools.md#burp-suite).
-  - [Frida via python-pip](0x08-Testing-Tools.md#frida)
-- At least one jailbroken iOS device (with the target iOS version installed).
-  - [Frida for iOS](0x08-Testing-Tools.md#frida-for-ios)
-- A Wi-Fi network that permits client-to-client traffic.
+- Ideally macOS host computer with admin rights
+- [Xcode](0x08-Testing-Tools.md#xcode) and [Xcode Command Line Tools](0x08-Testing-Tools.md#xcode-command-line-tools) installed.
+- Wi-Fi network that permits client-to-client traffic.
+- At least one jailbroken iOS device (of the desired iOS version).
+- [Burp Suite](0x08-Testing-Tools.md#burp-suite) or other interception proxy tool.
 
 ### Testing Device
 
-#### Testing on a Real Device
+#### Getting the UDID of an iOS device
+
+The UDID is a 40-digit unique sequence of letters and numbers to identify an iOS device. You can find the UDID of your iOS device on macOS Catalina onwards in the Finder app, as iTunes is not available anymore in Catalina. Just select the connected iOS device in Finder and click on the information under the name of the iOS device to iterate through it. Besides the UDID, you can find the serial number, IMEI and other useful information.
+
+<img src="Images/Chapters/0x06b/UDID-Finder.png" width="100%" />
+
+If you are using a macOS version before Catalina, you can find the [UDID of your iOS device via iTunes](https://medium.com/@igor_marques/how-to-find-an-iphones-udid-2d157f1cf2b9 "How to Find Your iPhone\'s UDID"), by selecting your device and clicking on "Serial Number" in the summary tab. When clicking on this you will iterate through different metadata of the iOS device including its UDID.
```

Comment:
Apart from the screenshot, the Medium article doesn't really add anything. Shouldn't we just create a screenshot ourselves?